spec/unit/document_spec.lua

```diff
@@ -72,20 +72,13 @@ describe("EPUB document module", function()
         assert.is_equal(doc:fastDigest(), "59d481d168cca6267322f150c5f6a2a3")
     end)
     it("should register droid sans fallback", function()
-        local fonts_registry = {
-            "Droid Sans Mono",
-            "FreeSans",
-            "FreeSerif",
-            "Noto Naskh Arabic",
-            "Noto Sans",
-            "Noto Sans Arabic UI",
-            "Noto Sans Bengali UI",
-            "Noto Sans CJK SC",
-            "Noto Sans Devanagari UI",
-            "Noto Serif",
-        }
-        local face_list = cre.getFontFaces()
-        assert.are.same(fonts_registry, face_list)
+        local has_droid_sans = false
+        for i, v in ipairs(cre.getFontFaces()) do
+            if v == "Droid Sans Mono" then
```

Comment:
I think there's also an implication that Droid Sans Mono is `i == 1` (fallback), and perhaps additionally that `#face_list >= 10` if the code is a truer representation of the intent than the `it("should…")`.

Reply:
Amended.